inference-engine/tests/functional/inference_engine/ngraph_reader/proposal_tests.cpp

```diff
@@ -306,100 +306,3 @@ TEST_F(NGraphReaderTests, ReadProposalNetwork_2) {
 
     compareIRs(model_v10, model_v6, 32);
 }
-
-TEST_F(NGraphReaderTests, ReadExtensionProposalNetwork) {
-    std::string model_v10 = R"V0G0N(
-<net name="Network" version="10">
-    <layers>
-        <layer id="0" name="in1" type="Parameter" version="opset1">
-            <data element_type="f32" shape="1,12,34,62"/>
```

Comment:
What if a user pass the same IR to IE 2021.3? Since it's IR V10 we should support it, but maybe not using generic op.
The same concern for IR with GRUCell below.

I think we need to keep this test, but adopt it to be:
```
... 
FAIL() << "Custom proposal layer is not a Generic operation!";
```
->
```
... 
FAIL() << "Custom proposal layer is a Generic operation!";
```

Reply:
Refactored tests + updated IR reader to load old "experimental" and "extension" ops as "opset6" ops.